wg-macros.toml

```diff
@@ -1,26 +1,39 @@
 name = "Rust Macros Working Group"
 description = "Meetings for the macros working group"
 
+[[timezones]]
+name = "Europe/Rome"
```

Comment:
Why add the timezone if it isn't being used?

Reply:
See https://github.com/rust-lang/calendar/blob/da8ee3c6ab0078ea1582f5f1cccb550a0b68a251/lang.toml#L15 as an example, it uses the timezones from https://github.com/rust-lang/calendar/blob/main/_timezones.toml (they're included at the top of the `lang.toml`)

Reply:
Thanks!